RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByIsolation.cc

```diff
@@ -39,6 +39,10 @@ class PFRecoTauDiscriminationByIsolation : public PFTauDiscriminationProducerBas
     calculateWeights_ = pset.exists("ApplyDiscriminationByWeightedECALIsolation") ?
       pset.getParameter<bool>("ApplyDiscriminationByWeightedECALIsolation") : false;
 
+    // RIC: allow to relax the isolation completely beyond a given tau pt
+    minPtForNoIso_ = pset.exists("minTauPtForNoIso") ?
```

Comment:
@veelken @mbluj @isobelojalvo @slava77 @rmanzoni 

Hi Slava, 

we will open a ticket to fix both issues here and in other places in a general clean-up task within the next days. What are our chances to get this specific PR integrated with this caveat? I just learned from Riccardo that it will be difficult for him to do it before for this specific case due to limited internet access this week. On the other hand if I understand him correctly he needs to meet a deadline with this PR basically tomorrow.